microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,258 @@
+
+
+##### [Group policy](#tab/group-policy)
+
+Set `BrowserExecutableFolder`.
+
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`
+Value: `<path to desired Microsoft Edge preview channel>`
+
+Do either of the following:
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  Click any of the three **Download Windows Policy** links.<!-- todo: no "Edge policy" download link there -->
+* Use the built-in policy on Intune.<!-- todo: link to more info -->
+
+---
+
+
+<!-- ====================================================================== -->
+## How to deploy non-stable channels
+
+Preview channels of Microsoft Edge are also called _insider channels_.  These are the non-stable channels of Microsoft Edge.
+
+You can deploy a preview channel of Microsoft Edge by doing either of the following:
+* Manually install preview channels on test machines.
+* Programmatically deploy preview channels via API.
+
+These options are explained below.
+
+
+<!-- ------------------------------ -->
+#### Option 1: Manually install preview channels on test machines
+
+Insider channels can be manually installed through the following links:
+
+| Channel | Link |
+| --- | --- |
+| Canary | [Download](https://go.microsoft.com/fwlink/?linkid=2084649&Channel=Canary) |
+| Dev | [Download](https://go.microsoft.com/fwlink/?linkid=2093291) |
+| Beta | [Download](https://go.microsoft.com/fwlink/?linkid=2093376) |
+
+This only needs to be done once per machine.  Prerelease channels are evergreen, so will automatically get updated when newer versions are available.
+
+
+<!-- ------------------------------ -->
+#### Option 2: Programmatically deploy preview channels via API
+
+<!-- todo: which api? -->
+
+In your app's code, periodically poll the following pages to get and deploy the latest version of each preview channel of Microsoft Edge.
+
+| Channel | Link |
+| --- | --- |
+| Canary | [https://edgeupdates.microsoft.com/api/products/canary](https://edgeupdates.microsoft.com/api/products/canary)<br>[MSI Link](https://go.microsoft.com/fwlink/?linkid=2084649&Channel=Canary) |
+| Dev | [https://edgeupdates.microsoft.com/api/products/dev](https://edgeupdates.microsoft.com/api/products/dev) |
+| Beta | [https://edgeupdates.microsoft.com/api/products/beta](https://edgeupdates.microsoft.com/api/products/beta) |
+
+The Edge Dev and Edge Beta channels contain MSI Links.  The Edge Canary channel has a separate MSI link.
+
+<!-- todo: See Also link? -->
+
+
+<!-- ====================================================================== -->
+## How to do automated testing
+
+See [Automate and test WebView2 apps with Microsoft Edge WebDriver](/microsoft-edge/webdriver).
```

Comment:
link shows 404 for me

Reply:
needs .md at end, i'll do this

Reply:
```
See [Automate and test WebView2 apps with Microsoft Edge WebDriver](./webdriver.md).
```
= https://learn.microsoft.com/microsoft-edge/webview2/how-to/webdriver